mix.exs

```diff
@@ -27,7 +27,8 @@ defmodule Protobuf.Mixfile do
       description: @description,
       package: package(),
       docs: docs(),
-      aliases: aliases()
+      aliases: aliases(),
+      xref: [exclude: Google.Protobuf.Any]
```

Comment:
This line appears to be causing the test build for Elixir 1.9.4 to fail. However, [looking at the docs](https://hexdocs.pm/mix/1.14.1/Mix.Tasks.Compile.Elixir.html#module-configuration), I just realized we're supposed to provide a _list of_ modules. Doing so seems to fix the issue for me…

```suggestion
      xref: [exclude: [Google.Protobuf.Any]]
```